lrauv_ignition_plugins/test/test_mission_depth_vbs.cc

```diff
@@ -85,19 +85,37 @@ TEST_F(LrauvTestFixture, DepthVBS)
   //       << std::endl;
   // }
 
-  this->CheckRange(2000, {0.01, -0.00, -0.78, -0.00, 0.04, 0.00}, {15.0, 3.14});
-  this->CheckRange(4000, {0.47, -0.00, -2.80, -0.00, 0.08, 0.00}, {15.0, 3.14});
-  this->CheckRange(6000, {2.77, -0.00, -5.73, 0.00, 0.12, 0.00}, {15.0, 3.14});
-  this->CheckRange(8000, {8.78, -0.04, -9.62, 0.00, 0.12, 0.03}, {15.0, 3.14});
-  this->CheckRange(10000, {15.66, 2.84, -13.07, 0.00, 0.09, 1.31}, {15.0, 3.14});
-  this->CheckRange(12000, {14.82, 8.71, -15.58, 0.00, 0.04, 2.55}, {15.0, 3.14});
-  this->CheckRange(14000, {10.69, 10.22, -16.95, -0.00, -0.03, -2.80}, {15.0, 3.14});
-  this->CheckRange(16000, {7.85, 8.85, -16.07, -0.00, -0.01, -2.13}, {15.0, 3.14});
-  this->CheckRange(18000, {6.48, 6.29, -13.84, 0.00, -0.07, -1.59}, {15.0, 3.14});
-  this->CheckRange(20000, {6.99, 2.26, -10.73, 0.00, -0.17, -0.93}, {15.0, 3.14});
-  this->CheckRange(22000, {11.22, -0.97, -7.71, -0.00, -0.10, 0.06}, {15.0, 3.14});
-  this->CheckRange(24000, {15.47, 0.32, -5.62, -0.00, 0.04, 0.98}, {15.0, 3.14});
-  this->CheckRange(26000, {16.95, 3.13, -5.16, 0.00, 0.00, 1.66}, {15.0, 3.14});
-  this->CheckRange(28000, {16.65, 5.57, -6.79, -0.00, 0.02, 2.17}, {15.0, 3.14});
+  ignition::math::Vector3d maxVel(0, 0, 0);
+  for (int i = 1; i <= this->tethysPoses.size(); i ++)
+  {
+    // Vehicle roll should be constant
+    EXPECT_NEAR(tethysPoses[i].Rot().Euler().X(), 0, 1e-2);
+
+    // Vehicle should not go vertical
+    EXPECT_LT(tethysPoses[i].Rot().Euler().Y(), IGN_DTOR(40));
+
+    // Vehicle should not exceed 20m depth
+    EXPECT_GT(tethysPoses[i].Pos().Z(), -20);
```

Comment:
Looking at the mission XML, the depth command is 10 m with a deadband of 0.1 m.

* Can we document that here?
* Can we make the expectation a bit tighter?

Reply:
So while the depth is 10m, it is not possible to cleanly descend to 10m, the controller will overshoot to about 16m before coming back to 10m as shown in the graph below. I believe this is consistent with the tank tests. The idea here is to manage the overshoot. I'll update the comment so its clearer.

![image](https://user-images.githubusercontent.com/542272/145710416-e3dbfdfa-b8a7-4a0d-92d9-08ed449ed800.png)

Reply:
Got it, makes sense. Then can we document that in a comment here? For future us looking at the test.